### DISCUSSION TITLE
I want to add turkish and japanese translation of reticulum.network website

### DISCUSSION THREAD

**Discussion by umutcamliyurt** [markqvist/Reticulum#413] [Requests for New Features]
Title says it all. Where should i send translated version of website

**Comment by markqvist** [comment]
Hello @N-E-M-E-S-1-S
Thank you so very much for offering your help with these translations! That is simply fantastic! The source files for the website are available here: https://github.com/markqvist/reticulum_website/tree/main/source

If you can just create new files with the corresponding language codes for each page, and translate the pages in that, it would be great. And also add translations for the menu items right here: https://github.com/markqvist/reticulum_website/blob/main/build.py#L37

You can send me the translated files any way you'd like; for example as a pull request on the website repository, or via Matrix or email.

Thank you so much for this!

**Reply by PoincareMesh** [reply]
I just added Dutch translations. I messed up a bit and ended up with multiple PR's instead of just one. But in the end I think they al should merge properly

**Reply by markqvist** [reply]
No problem! Thank you very much, that's great!
